_dev/src/components/smart-shopping-campaign-creation/smart-shopping-campaign-creation.vue

```diff
@@ -43,6 +43,14 @@
           </ol>
         </b-card-header>
         <b-card-body body-class="p-3 p-md-4">
+          <b-alert
+            v-if="$route.params.type === $options.CampaignTypes.PERFORMANCE_MAX"
+            show
+            variant="info"
+            data-test-id="unhandled-filters-alert"
+          >
+            {{ $t("smartShoppingCampaignCreation.alerts.PMaxExplanation") }}
```

Comment:
I'm wondering. Wouldn't it be helpful for merchants to have a link to the help tab in your message?